bolb_bot/cogs/events.py

```diff
@@ -68,14 +68,6 @@ async def on_command_error(self, ctx: Context, error: Exception):
                 color=self.bot.color,
             )
             ensure_future(ctx.send(embed=embed))
-            if ctx.guild is None:
```

Comment:
May want to remove "dev has been notified" from above

Reply:
> May want to remove "dev has been notified" from above

It does technically log the error and command to the console though, but what do you say?